language/en/interface.json

```diff
@@ -298,8 +298,8 @@
 		},
 		"keybinds": {
 			"title": "Keybinds",
-			"disclaimer": "These keybinds are set by default. If you remove/replace hotkey widgets, or use your own uikeys, they might stop working!",
-			"howtochangekeybinds":"To change them: in Settings/Control tab, set Keybindings to Custom to create the BAR/data/uikeys.txt file.\nEdit this file and type /keyreload in chat to reload them.",
+			"disclaimer": "These keybinds are set by default. If you switch to a keybind preset other than Grid, keys in the 3rd column will be inaccurate!",
```

Comment:
Wrong information is worse than no information. Either what is shown to the player is correct or it is not shown at all.

Reply:
Thank you for the review.

This update is meant to be better than what we have, but not perfect. I don't have the capabilities to make it "perfect." Can we do this as a band-aid until more capable people make the tools needed for "perfection?"

"Perfection" would require getActionHotkey to accept arguments "so you can distinguish between gridmenu_category 1 and gridmenu_category 2" (https://github.com/beyond-all-reason/Beyond-All-Reason/pull/3678#issuecomment-2332288577), and the ability to switch the display between gridmenu and the "old" non-gridmenu way of doing things - for the "Selecting build orders" part, top right, specifically. Both of those issues are beyond my capabilities.

Reply:
For the record, getActionHotkey needs to be able to accept from 0 to 2 arguments so that "gridmenu_key 1 1" also can be can be distinguished from "gridmenu_key 1 2" etc.

Reply:
Alternatively, I can remove the "Selecting build orders" and the "Group selection" parts. By that point, all information on the page would be correct.

Please confirm that I should just do that.